pom.xml

```diff
@@ -34,8 +34,8 @@
     <name>Gravitee.io - Cockpit - API</name>
 
     <properties>
-        <gravitee-bom.version>2.6</gravitee-bom.version>
-        <gravitee-node-api.version>1.26.0</gravitee-node-api.version>
+        <gravitee-bom.version>2.7</gravitee-bom.version>
+        <gravitee-node-api.version>2.0.0-bumpRxJava3-SNAPSHOT</gravitee-node-api.version>
```

Comment:
I guess we are waiting for the new version of `gravitee-node` to be released. But we don't have to forget to update it here